src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -499,11 +499,15 @@ private async Task CreateInternal(BlittableJsonReaderObject bjro, SubscriptionCr
             using (ServerStore.ContextPool.AllocateOperationContext(out TransactionOperationContext serverContext))
             using (serverContext.OpenReadTransaction())
             {
-                // need to wait on the relevant remote node
-                var node = Database.SubscriptionStorage.GetResponsibleNode(serverContext, name);
-                if (node != null && node != ServerStore.NodeTag)
+                var topology = ServerStore.GetClusterTopology(serverContext);
+                if (topology.AllNodes.Count != 1)
```

Comment:
Shouldn't we use Members here? cc @karmeli87

Reply:
this check is only to know if its single node cluster

Reply:
What will happened if you create a database in cluster of 3 nodes, but with replication factor of 1?
node A executes the command, but the database is created on node B.
Would that work as you expect?

Reply:
okay... this is the _cluster_ topology.